app/Resources/translations/default/ModulesBuybuttonliteAdmin.xlf

```diff
@@ -14,9 +14,9 @@ File: modules/ps_buybuttonlite/ps_buybuttonlite.php:49</note>
         <note>Context:
 File: modules/ps_buybuttonlite/ps_buybuttonlite.php:50</note>
       </trans-unit>
-      <trans-unit id="d401ff02b78a8ecc9b835168f697b6e7">
-        <source>Select Product</source>
-        <target>Select Product</target>
+      <trans-unit id="fa0e92ab3c39a4fbdf81351af8d377ef">
+        <source>Select a product</source>
```

Comment:
Cf. line 42, string is supposed to be localized in _Admin.Global_, it already exists there.

Reply:
I don't see it in Admin.Global 🤔

Reply:
I am talking about the string _Copy to clipboard_, on line 42. I wrote it here because I was not able to comment line 42. Sorry!

Reply:
👍